products/sle/main.pm

```diff
@@ -682,6 +682,9 @@ sub load_consoletests() {
         loadtest "console/zypper_ref";
         loadtest "console/yast2_lan" unless is_bridged_networking;
         loadtest "console/curl_https";
+        if (get_var('SCC_ADDONS', '') =~ /asmm/) {
```

Comment:
Your proposal doesn't work, we have all modules not only asmm, so I'll stick with my original version.